Main.java

```diff
@@ -4,6 +4,11 @@
 
     public static void main(String[] args) {
 
+        String myname="Olympew";
+        System.out.println(myname);
```

Comment:
nice job.  let's put some spaces in there for readability and variables should follow the naming convention of first word lowercase second word starting with uppercase letter.  String myName = "Olympew";